proto/cosmos/gov/v1beta2/gov.proto

```diff
@@ -8,7 +8,7 @@ import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 import "cosmos_proto/cosmos.proto";
 
-option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta2";
+option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
 
 // VoteOption enumerates the valid vote options for a given governance proposal.
 enum VoteOption {
```

Comment:
Not related to this PR, but flagging here: should we remove the deprecated `Vote.option` field?

If that sounds good, I can create an issue to track this.

Reply:
I actually thought that as well and wanted to ask. I think we should. But yes let's create an issue and do it in a follow up PR

Reply:
https://github.com/cosmos/cosmos-sdk/issues/10792